File: app/src/androidTest/java/skeletonlayout/RecyclerViewTest.java
package skeletonlayout;

import android.support.test.InstrumentationRegistry;
import android.support.test.filters.LargeTest;
import android.support.test.runner.AndroidJUnit4;
import android.support.v7.widget.RecyclerView;

import com.faltenreich.skeletonlayout.R;
import com.faltenreich.skeletonlayout.SkeletonLayoutUtils;

import org.junit.Test;
import org.junit.runner.RunWith;

@RunWith(AndroidJUnit4.class)
@LargeTest
public class RecyclerViewTest {

    @Test
    public void testSkeleton() {
        RecyclerView recyclerView = new RecyclerView(InstrumentationRegistry.getTargetContext());
        SkeletonLayoutUtils.applySkeletonAdapter(recyclerView, R.layout.list_item);
    }
}

File: app/src/androidTest/kotlin/com/faltenreich/skeletonlayout/RecyclerViewTest.kt
package com.faltenreich.skeletonlayout

import android.support.test.InstrumentationRegistry
import android.support.test.runner.AndroidJUnit4
import android.support.v7.widget.RecyclerView
import org.junit.Test
import org.junit.runner.RunWith

@RunWith(AndroidJUnit4::class)
class RecyclerViewTest {

    @Test
    fun useAppContext() {
        val recyclerView = RecyclerView(InstrumentationRegistry.getTargetContext())
        recyclerView.applySkeletonAdapter(R.layout.list_item)
    }
}
